modules/battenberg/1.1/battenberg.smk

```diff
@@ -50,11 +50,48 @@ _battenberg_CFG = CFG
 
 # Define rules to be run locally when using a compute cluster
 localrules:
+    _battenberg_get_refrence
```

Comment:
Should this be `_battenburg_get_reference`?

Reply:
I don't think so. I just copied the spelling of the word from from the file name

Reply:
If so it seems the filename has a typo too and should be changed. Unless "refrence" has a specific meaning here.

Reply:
It's not the file name it's the rule name, right? Battenberg (not battenburg) is correct. "Refrence" is the typo but both the rule and the line indicated would need to change to be consistent or this will break.